crates/core/liveview-native-core-swift/Sources/LiveViewNativeCore/Support.swift

```diff
@@ -0,0 +1,147 @@
+import Foundation
+
+class SimpleHandler: DocumentChangeHandler {
+    var callback: (Document, NodeRef) -> ()
+    init (
+        _ callback: @escaping (Document, NodeRef) -> ()
+    ) {
+       self.callback = callback
+    }
+    func handle(_ context: Document, _ changeType: ChangeType, _ node: NodeRef, _ parent: NodeRef?) {
+        switch changeType {
+        case .add:
+            self.callback(context, parent!)
+        case .remove:
+            self.callback(context, parent!)
+        case .change:
+            self.callback(context, node)
+        case .replace:
+            self.callback(context, parent!)
+        }
+    }
+
+}
+
+extension Document {
+    public subscript(ref: NodeRef) -> Node {
+        let data = self.get(ref)
+        return Node(self, ref, data)
+    }
+    public static func parseFragmentJson(payload: [String: Any]) throws -> Document {
+        let jsonData = try JSONSerialization.data(withJSONObject: payload, options: .prettyPrinted)
```

Comment:
Pretty printing seems unnecessary.
```suggestion
        let jsonData = try JSONSerialization.data(withJSONObject: payload)
```